doc/content/reference/email-templates/available.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Available Templates"
+description: ""
+weight: 3
+---
+
+The following email templates are currently used by The Things Stack:
```

Comment:
Maybe add notes to `pkg/identityserver/emails` to update this doc when adding templates and when adding fields to the structs?